flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CollectionFunctionsITCase.java

```diff
@@ -632,7 +632,8 @@ private Stream<TestSetSpec> arrayMaxTestCases() {
                                     LocalDate.of(2012, 5, 16),
                                     LocalDate.of(2092, 7, 19)
                                 },
-                                null)
+                                null,
+                                new Integer[] {1, 2})
                         .andDataTypes(
                                 DataTypes.ARRAY(DataTypes.INT()),
```

Comment:
comments:
why not add unit test
new Integer[]{}, because the flink's array constuctor function is different from spark
flink should contain at least 1 args.
spark can contain zero arg, for example array()
but they both can be empty array, for example, udf, user can constuct empty array by themself

Reply:
new Integer[]{}, test will report error.

Reply:
@hanyuzheng7 Whatever we cannot construct sql empty array in  the unit test